requirements/dev.txt

```diff
@@ -11,7 +11,7 @@ amqp==5.1.1
     #   -r requirements/base.txt
     #   kombu
 ansible==8.5.0
-    # via -r requirements/../deployment/requirements.in
+    # via -r /home/steven/Documents/VNG/open-zaak/deployment/requirements.in
```

Comment:
Hm...

Reply:
Hmm that's strange, don't remember doing anything different while compiling. I've reverted the changes